### DISCUSSION TITLE
Older versions of Wyze Cam V2 Firmware?

### DISCUSSION THREAD

**Discussion by Sadie33** [gtxaspec/wz_mini_hacks#326] [General]
Does anyone have copies of the stock Wyze Cam V2 firmware prior to those posted in 2022? I have an older device that won't boot with the latest firmware posted on the Wyze download and they removed all the previous versions of the firmware. I received this device in Christmas 2019 and its been working fine until the other day when it was flashed with the latest firmware and it now doesn't work. The dreaded yellow light problem. Any suggestion on how to get this device working again?

**Comment by bodgitandco** [comment]
I have copies of the following:-
demo_4.9.2.42
demo_4.9.2.52
demo_4.9.3.64
demo_4.9.4.28
demo_4.9.4.37
demo_4.9.6.156.bin

Any good good you ?

**Reply by kokitojorge-droid** [reply]
do you still have those files?

**Reply by bodgitandco** [reply]
Files long gone, have you tried Thingino   https://thingino.com/ way better and  easy to install.

**Reply by kohrar** [reply]
I still have some of the v2 firmware files. My other repo with all the older firmware was taken down by Wyze's DMCA notice. Try Thingino first. If it doesn't work, I can probably get what you need.

**Comment by kohrar** [comment]
Here're the v2 firmwares I've collected. Hope that's of use to you.
https://github.com/kohrar/Wyze-Firmwares/tree/master/v2

**Reply by Sadie33** [reply]
Thank you very much! I downloaded them and found that version 4.9.4.169 (September 24, 2019) works!!!

Now on to see if I can get wz_mini_hacks to work.

**Comment by ljmerza** [comment]
whats the latest version that will work with this hack?
